data_steward/retraction/retract_utils.py

```diff
@@ -79,6 +84,65 @@ def get_src_id(mapping_type):
     return src_id
 
 
+def get_datasets_list(project_id, dataset_ids):
+    # initialize list of all datasets in project
+    all_dataset_ids = [
+        dataset.dataset_id for dataset in bq.list_datasets(project_id)
+    ]
+
+    if not dataset_ids or dataset_ids == consts.NONE:
+        dataset_ids = []
+    elif dataset_ids == consts.ALL_DATASETS:
+        dataset_ids = all_dataset_ids
```

Comment:
This suggestion is optional.  I'd rather give too much feedback when running a retraction instead of too little.  You could add logging to both of these to give the person doing the retraction some more feedback.

```suggestion
    if not dataset_ids or dataset_ids == consts.NONE:
        dataset_ids = []
        logging.info("No datasets specified.  Defaulting to empty list.  Expect bucked only retraction.")
    elif dataset_ids == consts.ALL_DATASETS:
        dataset_ids = all_dataset_ids
        logging.info(f"All datasets are specified.  Setting dataset_ids to all datasets in project: {project_id}")
```